engine/event/env.go

```diff
@@ -0,0 +1,37 @@
+package event
+
+import (
+	"os"
+	"strconv"
+)
+
+const (
+	EnvKeyEventQueues  = "FLOGO_EVENT_QUEUES"
```

Comment:
Should probably be FLOGO_EVENT_QUEUE_SIZE, and corresponding methods and vars should also reflect that.

Reply:
@fm-tibco  Actually I follow the way we do for Worker and Queues. 
FLOGO_RUNNER_QUEUE
FLOGO_RUNNER_WORKERS

Do you think we should keep follow or change to FLOGO_EVENT_QUEUE_SIZE?  I agree that adding size more explicit.

Reply:
We should deprecate FLOGO_RUNNER_QUEUE, and make it FLOGO_RUNNER_QUEUE_SIZE and use FLOGO_EVENT_QUEUE_SIZE.  I think WORKERS is ok for now.